scripts/install-chef-suse.sh

```diff
@@ -1166,6 +1166,18 @@ for s in xinetd dhcpd apache2 ; do
     fi
 done
 
+# activate provisioner repos
+if ! is_ses; then
+    for i in SLES12-SP1-Pool SLES12-SP1-Updates SUSE-OpenStack-Cloud-6-Pool SUSE-OpenStack-Cloud-6-Updates SLES12-SP1-HA-Pool SLES12-SP1-HA-Updates; do
+      curl --digest -u crowbar:crowbar -X POST -d "platform=suse-12.1" -d "repo=$i" http://$IP/utils/repositories/activate
+    done
+fi
+
+if [ -e $MEDIA/boot/x86_64/common ]; then
```

Comment:
`MEDIA` can be set to different things at different point in time, might be best to be more explicit and use the SLE12 directory directly.